deploy/charts/arcadia/templates/agent-portal.yaml

```diff
@@ -76,7 +78,7 @@ spec:
   internalTrafficPolicy: Cluster
   ports:
   - name: http
-    port: 3000
+    port: 80
     protocol: TCP
```

Comment:
or we can use both 3000 for port and targetPort, and use 3000 directly in the ingress port number